packages/xo-server-audit/src/index.js

```diff
@@ -407,6 +426,74 @@ class AuditXoPlugin {
       }))
   }
 
+  async _handleImportRecords($defer, { fileStream, zipped }) {
+    $defer(await this._storage.acquireLock())
```

Comment:
this will acquire the lock at the end of this function, is it intended. And you'll need to release any lock taken